tests/CodeGenHelpers.Tests/SampleCode/MyComplexClass.cs

```diff
@@ -0,0 +1,7 @@
+namespace CodeGenHelpers.SampleCode
+{
+    public class MyComplexClass
+    {
+
+    }
```

Comment:
This doesn't appear to be used or complex 😉 

The SampleCode is used in-lieu of strings in the tests in order to keep the tests themselves cleaner.

Reply:
Sorry, this slipped in and clearly shouldn't be there :sweat_smile:. I only used it to do some local tests to make sure I didn't change any existing behaviour.